src/app/Page.js

```diff
@@ -39,7 +40,18 @@ export default class Page extends React.Component<{|+children: Node|}> {
                 >
                   <GithubLogo
                     altText="SourceCred Github"
-                    className={css(style.navLogo)}
+                    className={css(style.navLogoSm)}
+                  />
+                </a>
+              </li>
+              <li className={css(style.navItem, style.navItemRight)}>
+                <a
+                  className={css(style.navLink)}
+                  href="https://discordapp.com/invite/tsBTgc9"
+                >
+                  <DiscordLogo
+                    altText="Join the SourceCred Discord"
+                    className={css(style.navLogoMed)}
```

Comment:
I see why you made this 5px larger than the others—with the descender
from the speech bubble, it looks too small otherwise. This makes sense,
but has the side effect that the vertical alignment is different from
the others. Amusingly, the fix that looks best to me is 

```diff
diff --git a/src/app/Page.js b/src/app/Page.js
index 81150ec..b5ebdac 100644
--- a/src/app/Page.js
+++ b/src/app/Page.js
@@ -114,5 +114,6 @@ const style = StyleSheet.create({
   navLogoMed: {
     height: 25,
     width: 25,
+    transform: "translateY(-1px)",
   },
 });
```

So… while I feel silly for asking for a 1-pixel change, do you mind? :-)
(If you have a better way to implement the shift than a CSS transform,
go for it.)